### PR TITLE
[bn] Localized content/bn/auto_scaling.md

### PR DESCRIPTION
**Pull request by ikramulkayes**
Signed-off-Ikramul Kayes ikramul.kayesgg@gmail.com
Localizes ```content/bn/auto_scaling.md``` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #410

### CODE REVIEW

**Review by Mouly22 — APPROVED** [review]
Nicely done!

File: content/bn/auto_scaling.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং (Auto Scaling)
+status: Completed
+category: সম্পত্তি
+---
+
+অটোস্কেলিং হল একটি সিস্টেমের ক্ষমতা [scale](/scalability/) স্বয়ংক্রিয়ভাবে, সাধারণত, কম্পিউটিং সংস্থানগুলির পরিপ্রেক্ষিতে। একটি অটোস্কেলিং সিস্টেমের সাহায্যে, প্রয়োজনের সময় সংস্থানগুলি স্বয়ংক্রিয়ভাবে যোগ করা হয় এবং ব্যবহারকারীর অস্থির চাহিদা মেটাতে স্কেল করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তিত হয় এবং মেমরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। পরিচালিত ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং কার্যকারিতার সাথে যুক্ত থাকে কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে আরও বেশি বিকল্প এবং বাস্তবায়ন রয়েছে।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md:7]: @ikramulkayes Can you kindly check the sentence? It seems inappropriate

Comment (mitul3737) [reply content/bn/auto_scaling.md:7]: I also think `মেমোরি` would be a good use

File: content/bn/auto_scaling.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং (Auto Scaling)
+status: Completed
+category: সম্পত্তি
+---
+
+অটোস্কেলিং হল একটি সিস্টেমের ক্ষমতা [scale](/scalability/) স্বয়ংক্রিয়ভাবে, সাধারণত, কম্পিউটিং সংস্থানগুলির পরিপ্রেক্ষিতে। একটি অটোস্কেলিং সিস্টেমের সাহায্যে, প্রয়োজনের সময় সংস্থানগুলি স্বয়ংক্রিয়ভাবে যোগ করা হয় এবং ব্যবহারকারীর অস্থির চাহিদা মেটাতে স্কেল করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তিত হয় এবং মেমরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। পরিচালিত ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং কার্যকারিতার সাথে যুক্ত থাকে কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে আরও বেশি বিকল্প এবং বাস্তবায়ন রয়েছে। 
+
+পূর্বে, সর্বোচ্চ সিস্টেম ব্যবহার বিবেচনা করার জন্য অবকাঠামো এবং অ্যাপ্লিকেশনগুলিকে আর্কিটেক্ট করা হয়েছিল। এই স্থাপত্যের অর্থ হল যে আরও সংস্থানগুলি কম ব্যবহার করা হয়েছে এবং ভোক্তাদের চাহিদা পরিবর্তন করার জন্য অস্থিতিস্থাপক। অস্থিতিস্থাপকতা মানে ব্যবসার জন্য উচ্চ খরচ এবং অতিরিক্ত চাহিদার কারণে ব্যবসা বন্ধ হয়ে যাওয়া থেকে।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md:9]: Can you check this sentence too?

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@ikramulkayes , kindly use 1 branch to create just 1 file. In this branch, you have `canary_deployment.md` along with the `auto_scaling.md` file . Unless you take an action, we can not approve. I hope you understand.

**Comment by ikramulkayes** [comment]
> @ikramulkayes , kindly use 1 branch to create just 1 file. In this branch, you have `canary_deployment.md` along with the `auto_scaling.md` file . Unless you take an action, we can not approve. I hope you understand.

I hope it is okay now.

File: content/bn/auto_scaling.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং (Auto Scaling)
```
Comment (mitul3737) [inline content/bn/auto_scaling.md:2]: Can we have the title as `অটোস্কেলিং(Autoscaling)` ? For reference kindly check this [Glossary Page link](https://glossary.cncf.io/auto_scaling/)

File: content/bn/auto_scaling.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+status: Completed
+category: সম্পত্তি
+---
+
+অটোস্কেলিং হল সাধারণত একটি সিস্টেমের কম্পিউটিং রিসোর্সগুলির পরিপ্রেক্ষিতে স্বয়ংক্রিয়ভাবে  [স্কেল](/স্কেলযোগ্যতা/)  ক্ষমতা। অটোস্কেলিং সিস্টেমের সাহায্যে, ব্যবহারকারী চাহিদা অনুযায়ী রিসোর্স স্কেল করতে এবং প্রয়োজনের সময়  স্বয়ংক্রিয়ভাবে রিসোর্সগুলি যোগ করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তনশীল এবং মেমোরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং এর মাধ্যমে পরিচালিত হয় কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে এটিতে আরও বেশি বাস্তবায়ন করার ব্যবস্থা রয়েছে।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md:7]: I think you have localized the link `[স্কেল](/স্কেলযোগ্যতা/)`

File: content/bn/auto_scaling.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+পূর্বে, সিস্টেমের  অবকাঠামো এবং অ্যাপ্লিকেশনগুলিকে আর্কিটেক্ট করা হয়েছিল সিস্টেমের সর্বোচ্চ ব্যবহার বিবেচনা করার মাধ্যমে। এই স্থাপনার দ্বারা রিসোর্সগুলি কম ব্যবহার হচ্ছিল এবং ভোক্তাদের চাহিদা পরিবর্তন করার জন্য স্থিতিস্থাপক ছিল। এই স্থিতিস্থাপকতা ফলে ব্যবসায় উচ্চ খরচ ছিল এবং অতিরিক্ত চাহিদা দেখা দিলেই ব্যবসা বন্ধ হয়ে যাচ্ছিল।
+
+ক্লাউড, [virtualizing](/virtualization/), and [containerizing](/containerization/) অ্যাপ্লিকেশন এবং তাদের নির্ভরতা ব্যবহার করে, সংস্থাগুলি ব্যবহারকারীর চাহিদা অনুযায়ী অ্যাপ্লিকেশন তৈরি করতে পারে। তারা অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে তাদের স্কেল করতে পারে, একটি সর্বোত্তম ব্যবহারকারীর অভিজ্ঞতা প্রদান করে। যেমন প্রতি শুক্রবার সন্ধ্যায় Netflix-এর দর্শক সংখ্যা বৃদ্ধি। স্বয়ংক্রিয়ভাবে আউট করার অর্থ হল গতিশীলভাবে আরও সংস্থান যোগ করা: উদাহরণস্বরূপ, আরও ভিডিও স্ট্রিমিংয়ের অনুমতি দেয় এমন সার্ভারের সংখ্যা বৃদ্ধি করা এবং একবার ব্যবহার স্বাভাবিক হয়ে গেলে আবার স্কেল করা।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md:11]: Can you localize these words `virtualizing`, `containerizing` ?

Comment (mitul3737) [reply content/bn/auto_scaling.md:11]: I think this sentence can be modified `তারা অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে তাদের স্কেল করতে পারে, একটি সর্বোত্তম ব্যবহারকারীর অভিজ্ঞতা প্রদান করে`

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@ikramulkayes  sorry for the late response . Can you please check this updated file https://github.com/cncf/glossary/blob/main/content/en/auto-scaling.md and update your file?

File: content/bn/auto_scaling.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং(Autoscaling)
+status: Completed
+category: সম্পত্তি
+---
+
+অটোস্কেলিং হল সাধারণত একটি সিস্টেমের কম্পিউটিং রিসোর্সগুলির পরিপ্রেক্ষিতে স্বয়ংক্রিয়ভাবে  [স্কেল](/scalability/) ক্ষমতা। অটোস্কেলিং সিস্টেমের সাহায্যে, ব্যবহারকারী চাহিদা অনুযায়ী রিসোর্স স্কেল করতে এবং প্রয়োজনের সময়  স্বয়ংক্রিয়ভাবে রিসোর্সগুলি যোগ করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তনশীল এবং মেমোরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং এর মাধ্যমে পরিচালিত হয় কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে এটিতে আরও বেশি বাস্তবায়ন করার ব্যবস্থা রয়েছে।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md]:
```suggestion
অটোস্কেলিং হল সাধারণত একটি সিস্টেমের কম্পিউটিং রিসোর্সগুলির পরিপ্রেক্ষিতে স্বয়ংক্রিয়ভাবে  [স্কেল](/bn/scalability/) ক্ষমতা। অটোস্কেলিং সিস্টেমের সাহায্যে, ব্যবহারকারী চাহিদা অনুযায়ী রিসোর্স স্কেল করতে এবং প্রয়োজনের সময়  স্বয়ংক্রিয়ভাবে রিসোর্সগুলি যোগ করতে পারে। অটোস্কেলিং প্রক্রিয়া পরিবর্তনশীল এবং মেমোরি বা প্রক্রিয়া সময়ের মতো বিভিন্ন মেট্রিক্সের উপর ভিত্তি করে স্কেল করার জন্য কনফিগারযোগ্য। ক্লাউড পরিষেবাগুলি সাধারণত অটোস্কেলিং এর মাধ্যমে পরিচালিত হয় কারণ বেশিরভাগ অন-প্রিমাইজ ডিপ্লোয়মেন্টের চেয়ে এটিতে আরও বেশি বাস্তবায়ন করার ব্যবস্থা রয়েছে।
```

File: content/bn/auto_scaling.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+পূর্বে, সিস্টেমের  অবকাঠামো এবং অ্যাপ্লিকেশনগুলিকে আর্কিটেক্ট করা হয়েছিল সিস্টেমের সর্বোচ্চ ব্যবহার বিবেচনা করার মাধ্যমে। এই স্থাপনার দ্বারা রিসোর্সগুলি কম ব্যবহার হচ্ছিল এবং ভোক্তাদের চাহিদা পরিবর্তন করার জন্য স্থিতিস্থাপক ছিল। এই স্থিতিস্থাপকতা ফলে ব্যবসায় উচ্চ খরচ ছিল এবং অতিরিক্ত চাহিদা দেখা দিলেই ব্যবসা বন্ধ হয়ে যাচ্ছিল।
+
+ক্লাউড, [ভারচুয়ালাইজিং](/virtualization/), and [কন্টেইনারাইজিং](/containerization/) অ্যাপ্লিকেশন এবং তাদের নির্ভরতা ব্যবহার করে, সংস্থাগুলি ব্যবহারকারীর চাহিদা অনুযায়ী অ্যাপ্লিকেশন তৈরি করতে পারে। এটি অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে এসব অ্যাপ্লিকেশন স্কেল করতে পারে, যার ফলে এট ব্যবহারকারীদের সর্বোত্তম অভিজ্ঞতা প্রদান করে। যেমন প্রতি শুক্রবার সন্ধ্যায় Netflix-এর দর্শক সংখ্যা বৃদ্ধি পায়। স্বয়ংক্রিয়ভাবে আউট করার অর্থ হল গতিশীলভাবে আরও সংস্থান যোগ করা: উদাহরণস্বরূপ, আরও ভিডিও স্ট্রিমিংয়ের অনুমতি দেয় এমন সার্ভারের সংখ্যা বৃদ্ধি করা এবং একবার ব্যবহার স্বাভাবিক হয়ে গেলে আবার স্কেল করা।
```
Comment (mitul3737) [inline content/bn/auto_scaling.md]:
```suggestion
ক্লাউড, [ভারচুয়ালাইজিং](/bn/virtualization/), and [কন্টেইনারাইজিং](/bn/containerization/) অ্যাপ্লিকেশন এবং তাদের নির্ভরতা ব্যবহার করে, সংস্থাগুলি ব্যবহারকারীর চাহিদা অনুযায়ী অ্যাপ্লিকেশন তৈরি করতে পারে। এটি অ্যাপ্লিকেশন চাহিদা নিরীক্ষণ করতে পারে এবং স্বয়ংক্রিয়ভাবে এসব অ্যাপ্লিকেশন স্কেল করতে পারে, যার ফলে এট ব্যবহারকারীদের সর্বোত্তম অভিজ্ঞতা প্রদান করে। যেমন প্রতি শুক্রবার সন্ধ্যায় Netflix-এর দর্শক সংখ্যা বৃদ্ধি পায়। স্বয়ংক্রিয়ভাবে আউট করার অর্থ হল গতিশীলভাবে আরও সংস্থান যোগ করা: উদাহরণস্বরূপ, আরও ভিডিও স্ট্রিমিংয়ের অনুমতি দেয় এমন সার্ভারের সংখ্যা বৃদ্ধি করা এবং একবার ব্যবহার স্বাভাবিক হয়ে গেলে আবার স্কেল করা।
```

**Review by mitul3737 — APPROVED** [review]
LGTM

**Pull request merged**